README.md

```diff
@@ -84,16 +84,37 @@ If you use Visual Studio Code, it's recommended to install [Prettier - Code form
 ](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and add the following to your `settings.json`
 
 ```json
-"eslint.autoFixOnSave":  true,
-"eslint.validate":  [
-  "javascript",
-  "javascriptreact",
-  {"language":  "typescript",  "autoFix":  true  },
-  {"language":  "typescriptreact",  "autoFix":  true  }
-]
+"editor.codeActionsOnSave": {
+  "source.fixAll.eslint": true
+}
 ```
 
 ## Testnet faucets
 
 In order to get testing tokens, read up on the information here:
 [faucet](./docs/faucet-info.md)
+
+## Customizing components
+
+This app employs overrides from an alternative source (`custom/` directory) for easy swap in of imported files.
+
+Override works for absolute paths that make use of baseUrl
+That is if there are two files:
+
+```js
+src/pages/About.tsx
+custom/pages/About.tsx
+
+import About from 'pages/About'
```

Comment:
For that to work, we need to update our imports to be absolute from now on, no?

Reply:
I wouldn't go changing `import Header from './Header'` to `import Header from 'components/Layout/Header'`
But only because I don't want to.
You are right, to fully use `custom/ override` imports must be absolute